src/libsyntax/print/pprust.rs

```diff
@@ -1909,6 +1911,10 @@ impl<'a> State<'a> {
                 try!(self.pclose());
             }
             ast::ExprMac(ref m) => try!(self.print_mac(m, token::Paren)),
+            ast::ExprCompletion(ref e) => {
+                try!(self.print_expr(&**e));
+                try!(word(&mut self.s, ".\u{B6}"));
```

Comment:
Indeed, but both have relatively poor font support. Neither are supported by Courier New, the default monospaced font on Windows, for example.

By the way, what a bikeshed. :)